settings.gradle.kts

```
@@ -16,3 +16,4 @@
 rootProject.name = "radiography"
 
 include(":radiography")
+include(":sample")
```

Nit: Let's make this a list:
```suggestion
include(
  ":radiography",
  ":sample"
)
```